### PR TITLE
:wrench: create a titlePage for refactor

### PR DESCRIPTION
**Pull request by issa-diallo**

### CODE REVIEW

File: src/components/TitlePage.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  title: string
+}
+
+const TitlePage = ({ title }: titleProps): JSX.Element => {
+  return (
+    <div className='d-flex align-items-center justify-content-center py-4'>
+      <h1>{title}</h1>
+    </div>
+  )
+}
```
Comment (AndreMiras) [inline src/components/TitlePage.tsx:13]: Implicit return look neat don't they?
```
const TitlePage = ({ title }: titleProps): JSX.Element => (
  <div className='d-flex align-items-center justify-content-center py-4'>
    <h1>{title}</h1>
  </div>
)
}```

**Review by AndreMiras — APPROVED** [review]
Nice, left a minor comment

File: src/components/TitlePage.tsx (hunk 1 above)
Comment (issa-diallo) [reply src/components/TitlePage.tsx]: I'm thinking linter manage this

Comment (AndreMiras) [reply src/components/TitlePage.tsx]: hm really? :thinking:

**Pull request merged**